src/scripts/make-songs-includes.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+inputs=$(for f in `ls $SRC_DIR/songs/*.tex`; do echo "\input{songs/$(basename $f | cut -d. -f 1)}"; done)
```

Comment:
That's right. I think it's a feature and not a bug 😛 

I updated the migration script so it removes any leading "the-" from file names (not very elegant but we don't need to run that script anyway, since I've already done the migration).

Manual ordering can be done with file naming. I see two use cases:
- you want to test a song: you can prefix it's name with an underscore and it will automatically be included as the first song in the book. Once you're done, you can just rename the file and remove the underscore.
- you want songs from the same artist(s) (with different names) to be grouped, say The Police and Sting songs for instance. Well, you can use file names to group those. For instance song 1 could be in file `police_song-1` and song 2 would be `police-sting_song-2`. It's *only* the file name that would be affected. The tex code would still show "The Police" and "Sting" respectively.

Anyway, as you like 😺 

PS: I'm planning a future PR to improve the developing experience: build the PDFs with a subset of files, or just one for instance.